google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable/HelloWorldTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2018 Google LLC.  All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.examples.bigtable;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.google.api.gax.rpc.ServerStream;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminSettings;
+import com.google.cloud.bigtable.admin.v2.models.CreateTableRequest;
+import com.google.cloud.bigtable.admin.v2.models.Table;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.BigtableDataSettings;
+import com.google.cloud.bigtable.data.v2.models.InstanceName;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+import java.io.IOException;
+import org.junit.AfterClass;
+import org.junit.AssumptionViolatedException;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class HelloWorldTest {
+
+  private static final String INSTANCE_PROPERTY_NAME = "bigtable.instance";
+  private static final String TABLE_ID = "test-table" + System.currentTimeMillis();
+  private static final String COLUMN_FAMILY_ID = "test-cf";
+  private static final String COLUMN_QUALIFIER = "test-greeting";
+  private static final String ROW_KEY_PREFIX = "test-rowKey";
+  private static BigtableDataClient dataClient;
+  private static BigtableTableAdminClient adminClient;
+
```

Comment:
add a garbage collection step, something like:

I think it would be nicer to:
- add instance variables to HelloWorld for project id/instance id/etc
- make all methods non static
- remove per method parameters
- use main to simply instantiate HelloWord passing it the appropriate parameters and then call run

something like:
```java

  private static String generateTableId() {
    return String.format("%s-%016x-%d", TABLE_PREFIX, System.currentTimeMillis(), new Random().nextLong());
  }
  void garbageCollect() {
    Pattern timestampPattern = Pattern.compile(TABLE_PREFIX + "-([0-9]+");
    for(String tableId : tableAdminClient.listTables()) {
      Matcher matcher = timestampPattern.matcher(tableName.getNameAsString());
      if (!matcher.matches()) {
       continue;
      }
      String timestampStr = matcher.group(1);
      long timestamp = Long.parseLong(timestampStr);
      if (System.currentTimeMillis() - timestamp < TimeUnit.MINUTES.toMillis(15)) {
        continue;
      }
      System.err.println("Garbaging collecting orphaned table: " + tableId);
      tableAdminClient.deleteTable(tableId);
    }
  }
```